actionpack/lib/action_controller/metal/live.rb

```diff
@@ -66,6 +66,17 @@ def make_response!(request)
       end
     end
 
+    module Instrumentation
```

Comment:
Let's just inline this module. There is no need for this complex inheritance.

Reply:
Ah sorry @hannahramadan , this is what you had before 😞

Reply:
Yeah, I saw that I debated if we needed this and I'd not ask if we didn't have to change the PR to add a test.

Reply:
An easy edit! @p8 @rafaelfranca I've inlined this and added a test based on the suggestion below. I'll wait to squash until I receive feedback.